### PR TITLE
Release 1.2.0

### PR DESCRIPTION
**Pull request by noamblitz**

### CODE REVIEW

File: .github/workflows/container_image.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          # We don't use git context because that doesn't process .dockerignore
+          # https://github.com/docker/cli/issues/2827
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          secrets: |
+            "github_token=${{ secrets.RDO_REPO_RO }}"
```
Comment (underdarknl) [inline .github/workflows/container_image.yml:45]: Should this not just use the github_token used above?

File: .github/workflows/deb_package.yml [hunk 2]
```diff
@@ -34,67 +34,17 @@ jobs:
       - name: Replace version number with release version
         run: find . -type f -name 'version.py' -exec sed -ibak "s/__version__ = .*/__version__ = \"${RELEASE_TAG}\"/" {} \;
 
-      # TODO: proper deb package versions
-      - name: Create directories for .deb package
-        run: mkdir -p kat-boefjes_${{env.RELEASE_VERSION}}/{data/usr/share/kat-boefjes/app,debian}
+      - name: Clone octopoes
+        run: git clone https://${{ secrets.RDO_REPO_RO }}@github.com/minvws/nl-rt-tim-abang-octopoes.git /tmp/octopoes
```
Comment (underdarknl) [inline .github/workflows/deb_package.yml:38]: This should use the external public octopoes.

File: Dockerfile [hunk 3]
```diff
@@ -1,28 +1,41 @@
...
+ARG ENVIRONMENT
 
 COPY --from=boefjes-requirements /tmp/boefjes-requirements.txt /tmp/boefjes-requirements.txt
-RUN pip install -r /tmp/boefjes-requirements.txt
+COPY requirements-dev.txt .
 
-COPY nl-kat-boefjes/requirements-dev.txt .
-RUN pip install -r requirements-dev.txt
+RUN --mount=type=cache,target=/root/.cache --mount=type=secret,id=github_token \
+    git config --global url."https://`cat /run/secrets/github_token`@github.com/".insteadOf "ssh://git@github.com/" \
```
Comment (underdarknl) [inline Dockerfile:28]: This should also just use the public https repos without secrets..

**Review by underdarknl — CHANGES_REQUESTED** [review]

File: .github/workflows/container_image.yml (hunk 1 above)
Comment (noamblitz) [reply .github/workflows/container_image.yml]: After consulting @dekkers, decided to try without it. Seems to work.

File: Dockerfile (hunk 3 above)
Comment (noamblitz) [reply Dockerfile]: Better now?

**Review by underdarknl — APPROVED** [review]

**Pull request merged**